cf_units/__init__.py

```diff
@@ -1223,12 +1231,14 @@ def format(self, option=None):
                     bitmask |= i
             encoding = bitmask & \
                 (UT_ASCII | UT_ISO_8859_1 | UT_LATIN1 | UT_UTF8)
-            endocing_str = _encoding_lookup[encoding]
-            try:
-                result = _ud.format(self.ut_unit, bitmask)
-            except _ud.UdunitsError as e:
-                    self._propogate_error('Failed to format %r' % self, e)
-            return str(result.decode(endocing_str))
+            encoding_str = _encoding_lookup[encoding]
+            result = _ud.format(self.ut_unit, bitmask)
```

Comment:
Discussed with @pelson, removed unnecessary defensive `try...except`.

There were no test cases that raised this exception for `_ud.format`.